packages/lisk-p2p/src/index.ts

```diff
@@ -12,5 +12,8 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-export { P2P, IPeerReturnType, IPeerOptions } from './p2p';
-export { IPeerConfig, Peer } from './peer';
+export * from './errors';
+export * from './p2p';
+export * from './peer';
```

Comment:
I just realized, but probably in here only thing it needs to export is `p2p` and maybe `error` for library user